ooiui/static/js/views/common/NavbarView.js

```diff
@@ -60,8 +60,10 @@ var NavbarView = Backbone.View.extend({
     // Messages only appear to logged in users
     if(ooi.login.loggedIn()) {
         this.$el.find('#navbar-menus').append(this.templates.logged_in_nav_items());
+        this.$el.find('#Science').hide();
         //this.$el.find('#navbar-menus').append(this.messageView.el);
     }
     this.$el.find('#navbar-menus').append(this.dropdownUserView.el);
+   
   }
-});
```

Comment:
No, Andy was right.  I jsut wanted to make sure that automatic Red Flag was addressed.